modular_skyrat/modules/colony_fabricator/code/clothing/storage.dm

```diff
@@ -0,0 +1,88 @@
+// Backpacks
+
+/obj/item/storage/backpack/industrial/frontier_colonist
+	name = "frontier backpack"
+	desc = "A rugged backpack often used by settlers and explorers. Holds all of your equipment and then some."
+	icon = 'modular_skyrat/modules/colony_fabricator/icons/clothes/clothing.dmi'
+	icon_state = "backpack"
+	worn_icon = 'modular_skyrat/modules/colony_fabricator/icons/clothes/clothing_worn.dmi'
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
+	worn_icon_teshari = 'modular_skyrat/modules/colony_fabricator/icons/clothes/clothing_worn_teshari.dmi'
+	worn_icon_state = "backpack"
+	inhand_icon_state = null
+	// These have the storage of a duffelbag, however do not have the benefits that zipping/unzipping give a duffelbag (like bulky item storage)
+	storage_type = /datum/storage/duffel
+
+/obj/item/storage/backpack/industrial/frontier_colonist/satchel
+	name = "frontier satchel"
+	desc = "A rugged satchel often used by settlers and explorers. Holds less of your equipment than a backpack will."
+	icon_state = "satchel"
+	worn_icon_state = "satchel"
+	storage_type = /datum/storage/backpack
+
+/obj/item/storage/backpack/industrial/frontier_colonist/messenger
+	name = "frontier messenger bag"
+	desc = "A rugged messenger bag often used by settlers and explorers. Holds less of your equipment than a backpack will."
+	icon_state = "messenger"
+	worn_icon_state = "messenger"
+	storage_type = /datum/storage/backpack
+
+// Belts
+
+/obj/item/storage/belt/utility/frontier_colonist
+	name = "frontier chest rig"
+	desc = "A versatile chest rig with pockets to store really whatever you could think of within. \
+		That is, if whatever you could think of is within the realms of a utility belt. Fashion like this \
+		comes at a price you know!"
+	icon = 'modular_skyrat/modules/colony_fabricator/icons/clothes/clothing.dmi'
+	icon_state = "harness"
+	worn_icon = 'modular_skyrat/modules/colony_fabricator/icons/clothes/clothing_worn.dmi'
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
+	worn_icon_teshari = 'modular_skyrat/modules/colony_fabricator/icons/clothes/clothing_worn_teshari.dmi'
+	worn_icon_state = "harness"
+	inhand_icon_state = null
+
+/obj/item/storage/belt/utility/frontier_colonist/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/manufacturer_examine, COMPANY_FRONTIER)
+	atom_storage.max_slots = 7
+	atom_storage.max_specific_storage = WEIGHT_CLASS_NORMAL
+	// Can hold whatever a toolbelt can + some mining equipment for convenience
+	atom_storage.set_holdable(list(
```

Comment:
This belt feels overall pretty strong, it's basically just a better toolbelt, unless I'm missing something. Just feels like a big power creep again.